satpy/etc/readers/fci_l2_nc.yaml

```diff
@@ -2734,3 +2744,260 @@ datasets:
     file_type: nc_fci_asr
     file_key: product_timeliness
     long_name: product_timeliness_index
+
+# AMV Intermediate Product
+  intm_latitude:
+    name: intm_latitude
+    file_type: nc_fci_amvi
+    file_key: intm_latitude
+    standard_name: latitude
+
+  intm_longitude:
+    name: intm_longitude
+    file_type: nc_fci_amvi
+    file_key: intm_longitude
+    standard_name: longitude
+
+  intm_speed:
+    name: intm_speed
+    file_type: nc_fci_amvi
+    file_key: intm_speed
+    standard_name: wind_speed
+    coordinates:
+      - intm_longitude
+      - intm_latitude
+
+  intm_u_component:
+    name: intm_u_component
+    file_type: nc_fci_amvi
+    file_key: intm_u_component
+    standard_name: wind_speed_horizontal_component
+    coordinates:
+      - intm_longitude
+      - intm_latitude
+
+  intm_v_component:
+    name: intm_v_component
+    file_type: nc_fci_amvi
+    file_key: intm_v_component
+    standard_name: wind_speed_vertical_component
+    coordinates:
+      - intm_longitude
+      - intm_latitude
+
+  intm_direction:
+    name: intm_direction
+    file_type: nc_fci_amvi
+    file_key: intm_direction
+    standard_name: wind_to_direction
+    coordinates:
+      - intm_longitude
+      - intm_latitude
+
+  intm_pressure:
+    name: intm_pressure
+    file_type: nc_fci_amvi
+    file_key: intm_pressure
+    standard_name: wind_pressure
+    coordinates:
+      - intm_longitude
+      - intm_latitude
+
+  intm_temperature:
+    name: intm_temperature
+    file_type: nc_fci_amvi
+    file_key: intm_temperature
+    standard_name: wind_temperature
+    coordinates:
+      - intm_longitude
+      - intm_latitude
+
+  intm_target_type:
+    name: intm_target_type
+    file_type: nc_fci_amvi
+    file_key: target_type
+    standard_name: wind_target_type
+    coordinates:
+      - intm_longitude
+      - intm_latitude
+
+  intm_wind_method:
+    name: intm_wind_method
+    file_type: nc_fci_amvi
+    file_key: wind_method
+    standard_name: wind_wind_method
+    coordinates:
+      - intm_longitude
+      - intm_latitude
+
+# AMV Final Product
+  channel_id:
```

Comment:
This here looks like it should be metadata, but maybe I'm misunderstanding?

Reply:
Well I think this more of a legacy issue. 
Because AMVs are records and not 2D array there is a channel_id associated with each AMV element.
 It is I think because in BUFR and other binary format it is possible to have AMV from multiple channel in one record. Hence why it is treated as a variable here.

Reply:
Ok, that makes sense then. But should it be called `amv_channel_id` not to confuse with other products?

Reply:
Well that is again legacy, it is the name of the variable in the FS.